packages/unplugin/src/index.ts

```diff
@@ -3,6 +3,14 @@ import MagicString from "magic-string";
 import { getReleaseName } from "./getReleaseName";
 import { Options } from "./types";
 import { makeSentryFacade } from "./sentry/facade";
+import * as Sentry from "@sentry/node";
+import "@sentry/tracing";
+
+Sentry.init({
+  dsn: "https://1298ac2f87bc4aae99d9a9a94ce65b42@o447951.ingest.sentry.io/6681372",
```

Comment:
this dns is intentionally not an env var. I decided to keep it here as this is a public info and we should be transparent about this.

Reply:
Agree!